pkg/alerts/alertmanager/client.go

```diff
@@ -0,0 +1,118 @@
+package alertmanager
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+	"path"
+	"time"
+)
+
+const (
+	alertManagerPathPrefix = "/api/v2"
+	// the default context timeout for alert manager client
+	// feel free to change this value/set a corresponding env var if needed
+	defaultTimeOut = 10 * time.Second
+)
+
+// New is a constructor of AlertManagerClient
+func New() (*Client, error) {
+	if os.Getenv("ALERTMGR_URL") == "" {
+		return nil, fmt.Errorf("no alert manager url found")
+	}
+	return &Client{
+		Token:   os.Getenv("ALERTMGR_TOKEN"),
+		HostURL: os.Getenv("ALERTMGR_URL"),
+		Client:  new(http.Client),
```

Comment:
Please use cli-flags instead (they can be populated with env-variables as well)

Reply:
Perfect, thanks for feedback. That's exactly what I needed to know!

Reply:
Done